doc/design.txt

```diff
@@ -272,7 +272,7 @@ purposes; they are not actually present in the boot vector.
     -----------------+--------+--------'
      swap: none                        |
     -----------------------------------'
-    
```

Comment:
Sorry, I'm confused about this (and actually always have been).

This section says there are 5 states. The "BOOT STATES" section above says there are only 4. The code in boot_swap_tables now actually only has 3 entries.

What's going on here?